src/web.rs

```diff
@@ -35,9 +39,29 @@ impl WebDisplayImpl {
     }
 }
 
-pub struct WebImpl {
-    canvas: HtmlCanvasElement,
-    ctx: CanvasRenderingContext2d,
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(js_name = OffscreenCanvasRenderingContext2D)]
+    pub type OffscreenCanvasRenderingContext2d;
```

Comment:
This is was included in `web-sys` in https://github.com/rustwasm/wasm-bindgen/pull/3319, so I'd rather wait for that to be released than defining it ourselves.